routing/routing_serialization.hpp

```diff
@@ -121,22 +121,22 @@ class RestrictionSerializer
   /// \param end is an iterator to the element after the last element to serialize.
   /// \note All restrictions should have the same type.
   template <class Sink>
-  static void SerializeSingleType(routing::RestrictionVec::const_iterator begin,
-                                  routing::RestrictionVec::const_iterator end, Sink & sink)
+  static void SerializeSingleType(RestrictionVec::const_iterator begin,
+                                  RestrictionVec::const_iterator end, Sink & sink)
   {
     if (begin == end)
       return;
 
     CHECK(is_sorted(begin, end), ());
-    routing::Restriction::Type const type = begin->m_type;
+    Restriction::Type const type = begin->m_type;
```

Comment:
restriction type is the same for all elements in range [begin, end)? then it would be good to add corresponding assertion.